packages/nexus-sdk/src/View/types.ts

```diff
@@ -135,9 +137,78 @@ export type AggregatedSparqlViewPayload = {
   views: { project: string; viewId: string }[];
 };
 
+type RebuildStrategyType = {
+  '@type': 'Interval';
+  value: {};
+};
+
+export type CompositeView = Resource & {
+  '@id': ['CompositeView', 'Beta', 'View'];
+  _uuid: string;
+  sources: (
+    | ProjectEventStreamSource
+    | CrossProjectEventStreamSource
+    | RemoteProjectEventStreamSource)[];
+  projections: (Projection | ElasticSearchProjection)[];
+  rebuildStrategy?: RebuildStrategyType;
+};
+
+export type CrossProjectEventStreamSource = Source & {
+  '@type': 'CrossProjectEventStream';
+  project: string;
+  identities: {}[];
+};
+
+export type RemoteProjectEventStreamSource = Source & {
+  '@type': string;
```

Comment:
maybe use the same @type as found under Resources/types

Reply:
I have used 'RemoteProjectEventStream' string literal as the type since RemoteProjectEventStreamSource's type should be an exact string value.